src/se/mod.rs

```diff
@@ -100,6 +101,56 @@ pub fn to_string<S: Serialize>(value: &S) -> Result<String, DeError> {
     Ok(String::from_utf8(writer)?)
 }
 
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/// Defines which characters would be quoted in [`Text`] events and attribute
+/// values.
+///
+/// [`Text`]: Event::Text
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum QuoteLevel {
+    /// Performs quoting, escape all characters that could have special meaning
```

Comment:
Is "quoting" an official name for this?  I've only ever seen this referred to as escaping.

Reply:
I don't known. Replaced to "escaping"